### PR TITLE
Increase suite timeout to 30 minutes for docs tests

### PR DESCRIPTION
**Pull request by javanna**
I have been hitting the suite timeout on `DocsClientYamlTestSuiteIT`.
As far as I can see, the docs tests are taking quite a while, I assume
it's because more and more docs snippets get added over time, which
means more tests. The current suite timeout is the default 20 minutes.
It takes me just a little less than 20 minutes to run these on my
laptop. On my CI, I end up hitting the suite timeout. Hereby I propose
that we increase the suite timeout to 30 minutes.

### CODE REVIEW

**Comment by javanna** [comment]
cc @polyfractal @imotov

**Comment by jasontedor** [comment]
I think increasing to 30m is okay, but if we need to go any further we should investigate why these are taking so long rather than increasing the timeout even more. I suspect there is some environmental factor that we have not considered yet. I say this because these tests take 14m on my machine.

**Comment by javanna** [comment]
agreed @jasontedor that was my thinking too.

File: docs/src/test/java/org/elasticsearch/smoketest/DocsClientYamlTestSuiteIT.java [hunk 1]
```diff
@@ -48,12 +49,12 @@
...
 import java.util.Map;
 
-import static org.elasticsearch.common.xcontent.ConstructingObjectParser.constructorArg;
-
 import static java.util.Collections.emptyMap;
 import static java.util.Collections.singletonList;
 import static java.util.Collections.singletonMap;
+import static org.elasticsearch.common.xcontent.ConstructingObjectParser.constructorArg;
 
+@TimeoutSuite(millis = 30 * TimeUnits.MINUTE)
```
Comment (jasontedor) [inline docs/src/test/java/org/elasticsearch/smoketest/DocsClientYamlTestSuiteIT.java]: Can you leave a comment here summarizing our current thinking on this timeout value.

**Review by jasontedor — APPROVED** [review]
I left one comment. Otherwise, this LGTM.

File: docs/src/test/java/org/elasticsearch/smoketest/DocsClientYamlTestSuiteIT.java (hunk 1 above)
Comment (javanna) [reply docs/src/test/java/org/elasticsearch/smoketest/DocsClientYamlTestSuiteIT.java]: Something like "The default 20 minutes timeout is not always enough, please do not increase further than 30 minutes without analyzing what makes these tests so slow" ?

Comment (jasontedor) [reply docs/src/test/java/org/elasticsearch/smoketest/DocsClientYamlTestSuiteIT.java]: Yeah, something like that indeed. 😄

**Pull request merged**